static/templates/settings/deactivate_realm_modal.hbs

```diff
@@ -8,6 +8,6 @@
     </div>
     <div class="modal-footer">
         <button type="button" class="button rounded" data-dismiss="modal">{{t "Cancel" }}</button>
-        <button type="button" class="button rounded btn-danger" id="do_deactivate_realm_button">{{t "Deactivate organization" }}</button>
+        <button type="button" class="button rounded btn-danger" id="do_deactivate_realm_button">{{t "Confirm" }}</button>
```

Comment:
<img width="423" alt="deactivate_org_confirm_modal" src="https://user-images.githubusercontent.com/59444243/113509333-a4e1bd80-9572-11eb-9a9d-ed365d30637a.PNG">
Do we want to add a line here like: Are you sure you want to Deactivate this organisation?

Reply:
Yes, let's add: "Are you sure you want to deactivate this organisation?"

Reply:
Actually, for consistency with other confirmations, let's have it say:

"All users will permanently lose access to their Zulip accounts. This action cannot be undone.

Are you sure you want to deactivate this organisation?"